lib/services/ios-provision-service.ts

```diff
@@ -1,11 +1,19 @@
 import * as mobileprovision from "ios-mobileprovision-finder";
 import { createTable } from "../common/helpers";
 
-const months = ["Jan", "Feb", "Marc", "Apr", "May", "June", "July", "Aug", "Sept", "Oct", "Nov", "Dec"];
+const months = ["Jan", "Feb", "Mar", "Apr", "May", "Jun", "Jul", "Aug", "Sep", "Oct", "Nov", "Dec"];
 function formatDate(date: Date): string {
 	return `${date.getDay()} ${months[date.getMonth()]} ${date.getFullYear()}`;
 }
 
+/**
+ * Formats the argument so it can easily be copied from the terminal and provided as value for an option.
+ * @param arg The string to format.
+ */
+function cmdEscape(arg: string): string {
```

Comment:
No. It does cmdEscape. Its purpose is to make the string passable as argument to an option. Telerik A D can be escaped as "Telerik A D" or as Telerik\ A\ D and I wouldn't care less which one would be the result of the escape. On the other hand the escaping is flawed, it would fail on Telerik "AD" as the commas will go nuts with "Telerik "AD"" :( but I don't have time to figure out better escaping.